napari_assistant/_gui/_button_grid.py

```diff
@@ -41,6 +57,8 @@ def __init__(self, parent=None):
         self.setIconSize(QSize(64, 44))
         self.setWordWrap(True)
         self.setStyleSheet(STYLES)
+        self.setSpacing(4)
```

Comment:
I _think_ we may want to remoe this to restore the old layout. But I'm speculating...

Reply:
In the latest commit I changed the value slightly to match the old layout. You can see a screenshot in the latest comment!